scripts/zpool

```diff
@@ -47,28 +47,40 @@ This is a resource agent that manages ZFS zpools.
 EOF
 }
 
+zpool_status_log() {
+    # Log zpool status to splunk for post-mortem.  Run in it the background
```

Comment:
I put that comment in there since we're specificity logging with a `local7.debug` priority, which is one of the priorities that splunk will injest.  I'll remove it though.

Reply:
To make it more generic, the logging level should probably be an optional parameter to the resource agent (with a default).

Then again, to make this truly generic, we probably shouldn't be using logger at all, since ocf_log is the correct logging mechanism.  Really, we should probably be fixing our log collection to capture the ocf_log reports that we care about rather than hacking in an extra secondary logging mechanism.

But ultimately, this is a really minor point and I don't care. :)  If you guys want to move away from this being a generic agent and instead make it LLNL-specific, it is your prerogative to maintain it that way.

Reply:
We can probably alter our syslog-ng.conf to forward the zpool status output to the mgmt node, so it gets into splunk as well as into /var/log/pacemaker.log.